Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    add-apt-repository ppa:jonathonf/python-3.6 && \
+    add-apt-repository ppa:git-core/ppa && \
+    apt-get update && \
+    apt-get install -y python3.6 curl build-essential cmake libxml2-dev zlib1g-dev git && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && \
+    pip install conan && \
+    conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+
+COPY . /fbx2gltf
```

Comment:
Could this COPY be easily made to skip "./sdk/" by any chance? That's a lot of pointless data, if you're sure GIT LFS is not reliable everywhere.

Reply:
I should be able to add that to `.dockerignore`. The large copy wouldn't be an issue for docker hub builds because git lfs files haven't been pulled, but I can see this as useful for local builds.